tests/account_change_password_test.go

```diff
@@ -100,6 +101,83 @@ func TestChangePassword(t *testing.T) {
 			StatusCode: 200,
 		})
 	})
+
+	// sytest: Pushers created with a different access token are deleted on password change
+	t.Run("Pushers created with a different access token are deleted on password change", func(t *testing.T) {
+		sessionOptional := createSession(t, deployment, "test_change_password_user", "new_optional_password")
+		reqBody := client.WithJSONBody(t, map[string]interface{}{
+			"data": map[string]interface{}{
+				"url": "https://dummy.url/_matrix/push/v1/notify",
+			},
+			"profile_tag":         "tag",
+			"kind":                "http",
+			"app_id":              "complement",
+			"app_display_name":    "complement_display_name",
+			"device_display_name": "device_display_name",
+			"pushkey":             "a_push_key",
+			"lang":                "en",
+		})
+
+		res := sessionOptional.DoFunc(t, "POST", []string{"_matrix", "client", "r0", "pushers", "set"}, reqBody)
+
+		must.MatchResponse(t, res, match.HTTPResponse{
+			StatusCode: 200,
+		})
+
+		changePassword(t, passwordClient, "new_optional_password", newPassword)
+
+		res = passwordClient.DoFunc(t, "GET", []string{"_matrix", "client", "r0", "pushers"})
+		must.MatchResponse(t, res, match.HTTPResponse{
+			StatusCode: 200,
+			JSON: []match.JSON{
+				match.JSONKeyPresent("pushers"),
+				match.JSONArrayEach("pushers", func(val gjson.Result) error {
+					if len(val.Array()) != 0 {
+						return fmt.Errorf("expected array length to be zero: %v", val.Raw)
+					}
+					return nil
+				}),
+			},
+		})
+	})
+
+	// sytest: Pushers created with a the same access token are not deleted on password change
+	t.Run("Pushers created with a the same access token are not deleted on password change", func(t *testing.T) {
```

Comment:
Whilst there is a typo in the sytest name, please call `t.Run` with the correct name i.e

> Pushers created with the same access token are not deleted on password change